### DISCUSSION TITLE
Gsoc proposal

### DISCUSSION THREAD

**Discussion by Pawanupadhyay10** [polypheny/Polypheny-DB#326] [Q&A] [answered]
Hello, I have submitted my Gsoc2021 proposal, it is an request to my mentors, to kindly have a look on proposal and guide me so that I can improve it more, or please give me some valuable feedback.
Thanks.

**Comment by vogti** [comment, marked as answer]
Hi @Pawanupadhyay10 I've sent you a mail.

**Reply by Pawanupadhyay10** [reply]
Thanks , @vogti , for your suggestions , I have sent a mail to you please have a look .
